### PR TITLE
Patch

### PR DESCRIPTION
**Pull request by DataCrusade1999**
add: word and meaning
add: audio
add: phonetics

### CODE REVIEW

File: renderer.js [hunk 1]
```diff
@@ -38,5 +38,11 @@ async function getData(seachedWord) {
     // Extracting and storing JSON data from response
     const jsonData = await response.json();
 
-    console.log(jsonData);
+    console.log(jsonData[0]);
```
Comment (DataCrusade1999) [inline renderer.js:41]: will delete this line once i'm done

Comment (razorblack) [reply renderer.js:41]: Okay! No problem.

File: renderer.js [hunk 2]
```diff
@@ -38,5 +38,11 @@ async function getData(seachedWord) {
...
+    // Calling method to display data
+    document.getElementById("searched-word").innerHTML = seachedWord ;
+    document.getElementById("phonetics-text").innerHTML = jsonData[0].phonetics[0].text;
+    document.getElementById("phonetics-audio").src = 'https:' + jsonData[0].phonetics[0].audio;
+    document.getElementById("meaning-container1").innerHTML = jsonData[0].meanings[0].definitions[0].definition;
+    document.getElementById("meaning-container2").innerHTML = jsonData[0].meanings[1].definitions[0].definition;
```
Comment (razorblack) [inline renderer.js:47]: Put all DOM queries in separate function.

File: index.html [hunk 3]
```diff
@@ -56,13 +56,13 @@ <h3 id="app-subheading">A cross-platform on-click dictionary.</h3>
 
             <!-- Name of searched word -->
             <h3 id="searched-word"></h3>
+             <!-- Audio of phonetics -->
+             <audio controls id="phonetics-audio" src=""></audio>
```
Comment (razorblack) [inline index.html:60]: Put `<audio>` element at previous place.

File: index.html [hunk 4]
```diff
@@ -71,6 +71,18 @@ <h3 id="searched-word"></h3>
...
+
+            </div>
+
+            <div id="meaning-container2">
+
+            </div>
+
+            <div id="meaning-container3">
+
+            </div>
```
Comment (razorblack) [inline index.html:84]: * Rename inner container ids like `meaning-subcontainer1`
*  Use proper styling.

**Pull request closed without merging**